scripts/src/main/resources/scripts/command/aws

```diff
@@ -44,17 +56,22 @@ function doPackageInstall()
     powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${windows_path_to_package}\aws-${version}-windows.msi /quiet'"
   elif doIsMacOs
   then
-    doFail "Sorry, AWS CLI installation support is not yet implemented for your OS. Please install manually or help devonfw-ide to support it for your OS by contributing a pull-request."
+    doEcho "Installing AWS for MacOS..."
+    sudo installer -pkg "${path_to_package}"/aws-"${version}"-mac.pkg -target /
```

Comment:
If this command does not put the installation in the root folder of the hard drive `-target /`? I think we should put it in `/usr/local/bin` or something like that.

Reply:
This depends on the tar command. tar- x extrakt the files to usr/local/bin, so if I give / as target directory, it will put all files to /usr/local/bin/